stream_chat/async_chat/client.py

```diff
@@ -49,6 +49,14 @@ def __init__(
             connector=aiohttp.TCPConnector(keepalive_timeout=59.0),
         )
 
+    def set_http_session(self, session: aiohttp.ClientSession) -> None:
+        """
+        You can use your own `aiohttp.ClientSession` instance. This instance
+        will be used for underlying HTTP requests.
+        Make sure you set up a `base_url` for the session.
```

Comment:
do we need a similar instruction for the sync/requests version of this method?

Reply:
no, we use the full url there since it cannot be configured in `requests`